Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -865,7 +966,7 @@ Dynamic analysis of the app can show you which HTML or JavaScript files are load
 
 A full description of the attack is included in the [blog article by MWR](https://labs.mwrinfosecurity.com/blog/webview-addjavascriptinterface-remote-code-execution/ "WebView addJavascriptInterface Remote Code Execution").
 
-### Testing for Fragment Injection
+### Testing for Fragment Injection (MSTG-PLATFORM-2)
```

Comment:
maybe consider moving it next to ### Testing for Injection Flaws (MSTG‑PLATFORM‑2)

Reply:
makes sense. thanks